unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1675,55 +1679,15 @@ handleShowDefinition outputLoc showDefinitionScope query = do
   Backend.DefinitionResults terms types misses <- do
     let nameSearch = NameSearch.makeNameSearch hqLength names
     Cli.runTransaction (Backend.definitionsByName codebase nameSearch includeCycles Names.IncludeSuffixes (toList query))
-  outputPath <- getOutputPath
```

Comment:
Pulled out `showDefinition` into `ShowDefinition.hs` in line with our quest to shrink HandleInput.